packages/e2e/cypress/integration/ui/components/authenticator/auth-with-translations/auth-with-translations.steps.ts

```diff
@@ -0,0 +1,26 @@
+import { Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { translations } from '@aws-amplify/ui';
```

Comment:
I'm importing `translations` so that we have direct access to the expected translations without needing to copy and paste.